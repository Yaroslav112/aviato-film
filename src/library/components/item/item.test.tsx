import React from 'react';
import renderer from 'react-test-renderer';
import Item from './index';

const mockItems = [
  {
    index: 1,
    title: 'Some Name 1',
    poster_path: '/poster1.jpg',
  },
  {
    index: 2,
    title: 'Some Name 2',
    poster_path: '/poster2.jpg',
  },
];

test('Media Item renders correctly', () => {
  const tree = renderer.create(<Item items={mockItems} />).toJSON();
  expect(tree).toMatchSnapshot();
});
