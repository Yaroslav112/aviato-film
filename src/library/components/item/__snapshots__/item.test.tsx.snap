// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Media Item renders correctly 1`] = `
Array [
  <div
    className="sc-hKFymg klBRCO"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <img
      alt="Some Name 1"
      className="sc-gtsqUy bPbjIQ"
      src="https://image.tmdb.org/t/p/w300/poster1.jpg"
    />
    <p
      className="sc-dlniIP jzIpCH"
    >
      Some Name 1
    </p>
  </div>,
  <div
    className="sc-hKFymg klBRCO"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <img
      alt="Some Name 2"
      className="sc-gtsqUy bPbjIQ"
      src="https://image.tmdb.org/t/p/w300/poster2.jpg"
    />
    <p
      className="sc-dlniIP jzIpCH"
    >
      Some Name 2
    </p>
  </div>,
]
`;
